Mathlib/MeasureTheory/Integral/RieszMarkovKakutani.lean

```diff
@@ -36,6 +37,14 @@ variable (Λ : C_c(X, ℝ≥0) →ₗ[ℝ≥0] ℝ≥0)
 
 /-! ### Construction of the content: -/
 
+section Λ_mono
+
+lemma Λ_mono (f₁ f₂ : C_c(X, ℝ≥0)) (h : f₁.1 ≤ f₂.1) : Λ f₁ ≤ Λ f₂ := by
```

Comment:
What about
```suggestion
lemma CompactlySupportedContinuousMap.monotone_of_nnreal : Monotone Λ := by
```

Reply:
`LE` is not defined for `C_c(X, NNReal)`. for the moment I revert this.

Reply:
Can you leave a comment about why you can't spell that lemma with `Monotone`?